README.md

```diff
@@ -1,37 +1,103 @@
-# MarkLogic Unit Test
+![GitHub release](https://img.shields.io/github/release/marklogic-community/marklogic-unit-test.svg)
+![GitHub last commit](https://img.shields.io/github/last-commit/marklogic-community/marklogic-unit-test.svg)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+# Testing MarkLogic
 
-marklogic-unit-test includes the [original Roxy unit test framework for MarkLogic](https://github.com/marklogic-community/roxy/wiki/Unit-Testing) and 
-provides a few new features:
+marklogic-unit-test is an [ml-gradle bundle](https://github.com/marklogic-community/ml-gradle/wiki/Bundles) that allows
+a project to test MarkLogic code.  With one import a project immediately has access to:
 
-1. A REST endpoint for listing and running unit tests
-1. A small Java library for integrating MarkLogic unit tests into existing test frameworks like JUnit
-1. The ability to depend on this module as a true third-party dependency via ml-gradle
+1. A framework for writing and running MarkLogic unit tests, including several built in assertion functions
```

Comment:
I think these are good to call out, but I'm thinking it'd be nice to show something ASAP on this page. 

For example, after briefly describing marklogic-unit-test, hop into an example. Show a bare minimum library module (in SJS), like an "echo" function. Then show what the test module looks like. Then show a screenshot of running the test module via the UI. Then show a screenshot of running the test via Intellij (will require that skeleton Java class, may not be worth mentioning that yet).

That covers the main use case - testing library functions. A secondary use case is using marklogic-junit to test endpoints. I wouldn't go into detail about that on the home page though - instead, provide a link to a Wiki page (or the README for marklogic-junit). 

I think the screenshots of the UI and the Intellij (or Eclipse) test runner will carry the most value, because it immediately shows the user how they'll be running the tests.

Reply:
I created #81 to capture this request.  I think it's a good idea, but we need to get the landing page updated sooner than later.